src/main/java/uk/gov/hmcts/dts/fact/entity/Court.java

```diff
@@ -106,6 +106,10 @@
     @OneToMany(mappedBy = COURT_STRING, orphanRemoval = true)
     private List<CourtPostcode> courtPostcodes;
 
+    @OneToMany(mappedBy = COURT_STRING, orphanRemoval = true)
```

Comment:
Kind of a small one I think, so up to you - and probably not really a biggy. If we are not going to delete any links (it seems they are only being inserted), is it better to use persist, rather than all for the `orphanRemoval` value?